scripts-pipeline/fs.sh

```diff
@@ -47,7 +48,7 @@ set -eu
 
 get_if_remote t1
 
-if [[ $t1 != *nii || $t1 != *nii.gz ]]; then
+if [[ $t1 != *nii && $t1 != *nii.gz ]]; then
     tmpnii=/tmp/$(base "$t1").nii.gz
```

Comment:
This bug went unnoticed. Because of the bug, although you supply t1.nii or t1.nii.gz, it was still converting.